src/App.js

```diff
@@ -0,0 +1,21 @@
+import './App.css';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import Countries from './Components/countries';
+import Nav from './Components/Navigation';
+import Details from './Components/details';
+
+function App() {
+  return (
+    <>
+      <Nav />
+      <BrowserRouter>
+        <Routes>
+          <Route path="/" element={<Countries />} />
+          <Route path="/countryDetails/" element={<Details />} />
+        </Routes>
+      </BrowserRouter>
+    </>
+  );
+}
+
+export default App;
```

Comment:
- [ ] Nice job 👏 , however, your app does not render an interactive page. it's static with no links or navigation.
- Kindly review your component, making sure all paths and links are properly typed or inserted 👍 
- [ ] Follow [This link](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md) to see the complete list of requirements

### Your app on render:
![image](https://github.com/unorjikingsley/Metrics-WebApp/assets/50990925/54693a2f-9c8b-4d44-a6e8-f6983389c3c6)

### Requirments:
![image](https://github.com/unorjikingsley/Metrics-WebApp/assets/50990925/5048269f-8b9e-4926-91b1-d1e5c5e76ed5)